scripts/16_set-deposit-limit.js

```diff
@@ -0,0 +1,23 @@
+const {
+    callTokenManagerFunction,
+} = require('./helpers/10-call-token-manager-fn');
+
+async function setDepositLimit(minOrMax, newLimit) {
```

Comment:
Not critical, but maybe helpful to set the type of minOrMax to be ``'min' | 'max'``

Reply:
this is JS not TS, but you would be right!

Reply:
Thank you!